FinHubSA/information-storage-and-backend
# feat: add get all articles endpoint

# Pull Request

## Scope
https://dev.azure.com/CPerrie/Aarons%20Kit%20-%20Information%20Storage/_workitems/edit/47
## Work Done
Changed journals.py to articles.py.
Changed the app route  to "/articles"
Updated the SQL statements so that it outputs all the articles in the MySQL database.
## Steps to Test
Run the app through docker (docker compose up)
type 'localhost:5000/articles' in your web browser
The metadata of the dummy articles should appear (see example below).
<img width="857" alt="metadata" src="https://user-images.githubusercontent.com/68713894/132659452-6c4bc21a-b1b5-4fbe-bfb3-521d17179248.png">
